src/ext.js

```diff
@@ -201,12 +201,12 @@ export default function ext({ translator, shouldHide, senseNavigation, theme, is
                     }));
                   },
                 },
-                chart: {
+                chartId: {
                   type: "string",
                   expression: "optional",
-                  ref: "navigation.chart",
-                  translation: "properties.chartPicker.chart",
-                  show: (data) => (isGoToChartEnabled ? checkShowNavigation(data, "chart") : false),
+                  ref: "navigation.chartId",
```

Comment:
Are there any sn-action-button that already exist with the `navigation.chart` reference? If so, is there not a risk this will break those charts?

Reply:
No, not any more reference, this s a new feature, the other part of it that makes it work in in sense-client [here](https://github.com/qlik-trial/sense-client/pull/25579)